path to config.yml file issue

Dear Dr Jose,
I am trying to replicate your work but getting error for path to config.yml error. 
would you be able to provide the extra steps to include path to yml files.  
thank you.

Dear Dr @gante, 
![image](https://user-images.githubusercontent.com/78262128/157423741-13a74765-a6d4-4cd2-acc7-6321d82a7e01.png)
please take a look to the screen shot which says: 
0it [00:00, ?it/s]
Traceback (most recent call last):
  File "test_model.py", line 165, in <module>
    main()
  File "test_model.py", line 133, in main
    "The predictions and the labels must have the same number of examples!"
AssertionError: The predictions and the labels must have the same number of examples!

--- a/bff_positioning/data/dataset_handling.py
+++ b/bff_positioning/data/dataset_handling.py
@@ -109,7 +109,7 @@ def _get_scaler(scaler_type):
     """
     scaler = None
     if scaler_type == "binarizer":
-        scaler = Binarizer(0.1, copy=False)
+        scaler = Binarizer(threshold=0.1, copy=False)
     elif scaler_type == "normalizer":
         scaler = Normalizer(copy=False)
     elif scaler_type is not None:

--- a/bin/test_model.py
+++ b/bin/test_model.py
@@ -118,10 +118,10 @@ def main():
         if not mc_dropout_samples: # MC Dropout OFF
             predictions_test = model.predict(features_test)
             y_pred.append(predictions_test)
-
-        for sample_rnd in tqdm(range(mc_dropout_samples)): # equivalent to else, MC Dropout ON
-            predictions_test = model.predict(features_test)
-            y_pred.append(predictions_test)
+        else: # MC Dropout ON
+            for sample_rnd in tqdm(range(mc_dropout_samples)):
+                predictions_test = model.predict(features_test)
+                y_pred.append(predictions_test)
 
     # Stack results and sanity check
     y_true = np.vstack(y_true)
@@ -129,9 +129,9 @@ def main():
         y_pred = np.stack(y_pred, axis=2)
     else:
         y_pred = np.vstack(y_pred)
-    assert labels_test.shape[0] == y_true.shape[0] == y_pred.shape[0], \
+    assert y_true.shape[0] == y_pred.shape[0], \
         "The predictions and the labels must have the same number of examples!"
-    assert labels_test.shape[1] == y_true.shape[1] == y_pred.shape[1], \
+    assert y_true.shape[1] == y_pred.shape[1], \
         "The number of dimensions per sample must stay constant!"
 
     # Closes the model, gets the test scores, and stores predictions-labels pairs

